### PR TITLE
[release_15] Release 15 Merge into Master

### PR DESCRIPTION
**Pull request by ervteng**
### Proposed change(s)

Merge Release 15 back into `main`. 

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/main/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/main/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Installation-Anaconda-Windows.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ pip will get stuck when trying to read the cache of the package. If you see
 this, you can try:
 
 ```console
-python -m pip install mlagents==0.24.1 --no-cache-dir
+python -m pip install mlagents==0.26.0.dev0 --no-cache-dir
```
Comment (surfnerd) [inline docs/Installation-Anaconda-Windows.md]: are these right?

File: docs/Installation.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ To install the `mlagents` Python package, activate your virtual environment and
 run from the command line:
 
 ```sh
-python -m pip install mlagents==0.24.1
+python -m pip install mlagents==0.26.0.dev0
```
Comment (surfnerd) [inline docs/Installation.md]: same here

**Review by surfnerd — APPROVED** [review]
:shipit:

File: docs/Installation.md (hunk 2 above)
Comment (ervteng) [reply docs/Installation.md]: Yeah reverted these...

File: com.unity.ml-agents/CHANGELOG.md [hunk 3]
```diff
@@ -6,33 +6,45 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+# Changelog
```
Comment (chriselion) [inline com.unity.ml-agents/CHANGELOG.md:9]: This got duplicated

**Pull request merged**